test/unit/specs/components/staking/__snapshots__/TabValidators.spec.js.snap

```diff
@@ -392,392 +14,6 @@ exports[`TabValidators shows a message if still connecting 1`] = `
 </div>
 `;
 
-exports[`TabValidators shows a message if still loading 1`] = `
```

Comment:
looks like we're not showing the loading message

Reply:
good catch